### PR TITLE
S1task2

### PR DESCRIPTION
**Pull request by Mead-y**
HTML, CSS and some PHP for story 1

### CODE REVIEW

File: animals.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+$query->execute();
+$allAnimals = $query->fetchAll();
+//echo '<pre>';
+//var_dump($allAnimals);
+//echo '</pre>'
+
+
+//    echo '<pre>';
+//var_dump($animals['name']);
+//echo '</pre>';
```
Comment (haefnir) [inline animals.php]: This should probs be deleted if it isnt doing anything

File: animals.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+?>
+
+
+    <!DOCTYPE html>
+    <html lang="en-GB">
+    <head>
+        <title>YWP Collection App</title>
+        <link rel="stylesheet" href="index.css" type="text/css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
```
Comment (haefnir) [inline animals.php]: would define the charset here

File: animals.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    ?>
+    <img src= <?php echo $animal['image']; ?>
+                 alt = "Animal Image">
+            <h2 > Name: <?php echo $animal['name']; ?> </h2>
+            <h2 > Status: <?php echo $animal['status']; ?> </h2>
+            <h2 > Weight: <?php echo $animal['weight_kg']; ?> KG </h2>
+            <h2 > Threat: <?php echo $animal['threat']; ?> </h2>
+            <h2 > Diet: <?php echo $animal['diet']; ?> </h2>
+            <h2 >  <?php echo $animal['distribution']; ?> </h2>
+        </div >
```
Comment (haefnir) [inline animals.php]: could this be a function?

File: index.css [hunk 4]
```diff
@@ -0,0 +1,61 @@
+* {
+    margin: auto;
+    box-sizing: border-box;
+    font-family: MyriadTilt;
```
Comment (haefnir) [inline index.css:4]: Consider setting fallback font-family

File: animals.php [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    <main>
+<?php
+if (!$allAnimals) {
+    echo "You have not collected any animals yet!";
+}
+foreach ($allAnimals as $label => $animal) {
+    echo "<div class='animals'>";
+    ?>
+    <img src= <?php echo $animal['image']; ?>
+                 alt = "Animal Image">
```
Comment (haefnir) [inline animals.php]: could the alt text be the name of the animal?

**Review by haefnir — CHANGES_REQUESTED** [review]
Looking good! A few minor changes I reckon

File: test/displayAnimalsTest.php [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    public function testFailureDisplayAnimal(){
+        $input = 'hello';
+
+        $expected = "<div class='animals'><img src='image' alt = 'Polar Bear'><h2> Name: Polar Bear<h2> Status: Vulnerable Species<h2> Weight: 600 KG<h2> Threat: Climate Change<h2> Diet: Carnivore<h2> Arctic</div>";
+        $case = displayAnimals($input);
+        $this -> assertEquals($expected, $case);
+    }
+}
+
+class displayAnimalsTest extends Testcase {
```
Comment (ashleycoles) [inline test/displayAnimalsTest.php]: All of your test methods here should be in a single displayAnimalsTest class rather that duplicating the class. Worth re-running the tests after you do this just incase!

**Review by ashleycoles — CHANGES_REQUESTED** [review]

**Pull request merged**